pom.xml

```diff
@@ -75,23 +75,40 @@
       <artifactId>jackson-databind</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.datatype</groupId>
+      <artifactId>jackson-datatype-jsr310</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-parameter-names</artifactId>
+      <version>${jackson.version}</version>
+    </dependency>
     <dependency>
       <groupId>com.squareup.okhttp3</groupId>
       <artifactId>okhttp</artifactId>
       <version>${okhttp.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-api</artifactId>
-      <version>${junit.version}</version>
-      <scope>test</scope>
+      <groupId>org.eclipse.jgit</groupId>
+      <artifactId>org.eclipse.jgit</artifactId>
+      <version>6.9.0.202403050737-r</version>
     </dependency>
+
+    <!-- Autograder -->
     <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <version>${junit.version}</version>
-      <scope>test</scope>
+      <groupId>de.firemage.autograder</groupId>
+      <artifactId>autograder-core</artifactId>
+      <version>0.5.41</version>
     </dependency>
+    <dependency>
+      <groupId>de.firemage.autograder</groupId>
+      <artifactId>autograder-extra</artifactId>
+      <version>0.5.41</version>
+    </dependency>
```

Comment:
Please add a property for the version

Reply:
Should `autograder-extra` be included in `artemis4j`? I think one can exclude dependencies, but is this desired?

Reply:
I would say so, since artemis4j is not going to be used in the Test Framework